services/autoscaling/tests/manual/docker-compose.yml

```diff
@@ -48,7 +48,8 @@ services:
         scheduled_maintenance:${REDIS_HOST}:${REDIS_PORT}:3,
         user_notifications:${REDIS_HOST}:${REDIS_PORT}:4,
         announcements:${REDIS_HOST}:${REDIS_PORT}:5,
-        distributed_identifiers:${REDIS_HOST}:${REDIS_PORT}:6
+        distributed_identifiers:${REDIS_HOST}:${REDIS_PORT}:6,
+        distributed_caches:${REDIS_HOST}:${REDIS_PORT}:7
```

Comment:
please revert that one. it is unnecessary and I will remove that stuff anyway from there. autoscaling uses redis in a very simple manner and does not need this. thanks and sorry for the confusion